### ISSUE TITLE
Saving/loading workspace

### ISSUE THREAD

**Issue by igorrass** [mattwhitfield/VisualFindReferences#39]
I think it would be quite helpful if I could save the graph and load it back later, synching it with the new changes.
Also, it seems right now there is only one graph available, so I lose it when I switch to another tree. Would be nice to be able to open multiple tabs

**Comment by mattwhitfield** [comment]
I agree, that sounds like a good addition. It would be a fairly significant change, however. I don't have a lot of time at the moment, but hopefully I will be able to get to it soon.
